cards-test.py

```
@@ -0,0 +1,46 @@
+# presidio_smoke.py — noisy samples to exercise Microsoft Presidio–style detectors.
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)